templates/form/input--checkbox.html.twig

```diff
@@ -11,4 +11,10 @@
  */
 #}
 
-<input{{ attributes.addClass('ecl-checkbox__input').addClass('ecl-u-sr-only') }} />{{ children }}
+{% include '@ecl/ecl-forms-checkboxes' with {
+    'id': attributes.id,
+    'name': attributes.name,
+    'value': attributes.value,
+    'checked': attributes.checked,
+    'label': title
+} %}
```

Comment:
According to https://ec-europa.github.io/europa-component-library/components/detail/ecl-forms-checkboxes, some parameters are missing:
    - "is_disabled" (boolean): is disabled (optional, default: false)
    - "has_error" (boolean): has error (optional, default: false)
    - "extra_class" (string): extra CSS classes (optional, default: '')